### PR TITLE
Strings, Arrays and Safe-Box

### PR DESCRIPTION
**Pull request by maiderhernandorena**
Strings, Arrays and Safe-box

### CODE REVIEW

File: staff/maider-hernandorena/arrays-native/scripts/concat.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    expect(arrayLetters.length).toBe(5);
+    expect(arrayLetters[0]).toBe('a');
+    expect(arrayLetters[1]).toBe('b');
+    expect(arrayLetters[2]).toBe('c');
+    expect(arrayLetters[3]).toBe('d');
+    expect(arrayLetters[4]).toBe('e');
+
+
+    expect(result).toEqual([1, 2, 3, 4, 5, 'a', 'b', 'c', 'd', 'e'])
```
Comment (Javier1177) [inline staff/maider-hernandorena/arrays-native/scripts/concat.test.js]: Semicolon missed!

File: staff/maider-hernandorena/arrays-native/scripts/join.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+describe('Array.prototype.join', function () {
+    var array ;
```
Comment (Javier1177) [inline staff/maider-hernandorena/arrays-native/scripts/join.test.js]: You have a space between semicolon and the variable.

**Review by Javier1177 — APPROVED** [review]

File: staff/maider-hernandorena/spotify-app/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (jaumeserr) [inline staff/maider-hernandorena/spotify-app/index.html]: You need to put a name into title tag. No more 'Document', please. It hurts me...

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/main.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+// my presentation logic
```
Comment (jaumeserr) [inline staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/main.js:3]: you should use 'use strict' at the beginning of the js page.

File: staff/maider-hernandorena/spotify-app/scripts/logic.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+var logic = {
```
Comment (jaumeserr) [inline staff/maider-hernandorena/spotify-app/scripts/logic.js]: you should use 'use strict' at the beginning of the js page.

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/my-components.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
+// my custom components
```
Comment (jaumeserr) [inline staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/my-components.js:3]: you should use 'use strict' at the beginning of the js page.

File: staff/maider-hernandorena/arrays-native/scripts/concat.test.js (hunk 1 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/arrays-native/scripts/concat.test.js]: Changed! :D

File: staff/maider-hernandorena/arrays-native/scripts/join.test.js (hunk 2 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/arrays-native/scripts/join.test.js]: Changed! :D

File: staff/maider-hernandorena/spotify-app/index.html (hunk 3 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/spotify-app/index.html]: Changed! Sorry, i didn't want to hurt you ... :D

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/main.js (hunk 4 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/main.js:3]: Changed in all files! :D

File: staff/maider-hernandorena/spotify-app/scripts/logic.js (hunk 5 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/spotify-app/scripts/logic.js]: Changed in all files! :D

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/my-components.js (hunk 6 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/my-components.js:3]: Changed in all files! :D

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js [hunk 7]
```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+/**
+ * Web Components lib
+ * 
+ * @author manuelbarzi
```
Comment (SergioCastilloSorribes) [inline staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js:6]: Ohhh my god!!! This code was made by Manuel Barzi, not you!!!

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+/**
+ * Constructs a list of items
+ * 
+ * @param {*} array The item values
+ */
+function List(array) {
+    Component.call(this, 'ul');
+
+    // for (var i in array) {
```
Comment (SergioCastilloSorribes) [inline staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js:99]: ¿Have you left all the comments in the code?

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js (hunk 7 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js:6]: Yes i know it ! That is his version. My version is the 0.0.1. However, should i change the author name?

File: staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js (hunk 8 above)
Comment (maiderhernandorena) [reply staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/web-components-1.0.0.js:99]: i have left them commented to see different ways of doing the same code! :D

**Pull request merged**